### ISSUE TITLE
GUI bug driving me crazy :)

### ISSUE THREAD

**Issue by jeanfbrito** [4refr0nt/ESPlorer#48]
If the tooltip is summoned one time, all the GUI get weird and each button redraws the window. Take a look:
![image](https://cloud.githubusercontent.com/assets/1177482/21261271/70dabd70-c371-11e6-88fb-e7556cacf711.png)
Any way to disable tooltips?

**Comment by 4refr0nt** [comment]
Sorry, but you can disable each tooltip only in source code

**Comment by jeanfbrito** [comment]
Yes, I'm installing the NetBeans to do that. Any idea of what is causing that bug?

**Comment by 4refr0nt** [comment]
I can't reproduce this bug

**Comment by jeanfbrito** [comment]
There is a bug with multiple monitors, I think that is it. Do you use more than a monitor?

**Comment by 4refr0nt** [comment]
no

**Comment by jeanfbrito** [comment]
HA! Take a look!
All the tooltips are working well, and they are yellow:
![image](https://cloud.githubusercontent.com/assets/1177482/21262395/496f240e-c378-11e6-9d92-01284657b55a.png)
But, the AT commands line tooltip is of another cooler, gray, and ONLY WHEN this tooltip is evoked, that the interface bugs.
![image](https://cloud.githubusercontent.com/assets/1177482/21262424/70c73122-c378-11e6-80ca-622e2800c8db.png)

If I remove the text from it, the bug stops. Then, there is a problem with this component.
Is there something so different in it? The "send" button, on the right, have this problem too.

**Comment by jeanfbrito** [comment]
The color changes when the component is disabled. The color is not the problem. The problem is on the "command" to send area.

**Comment by jeanfbrito** [comment]
Just removing the "Command to send" tooltip resolved my problem. Weird not?

**Comment by 4refr0nt** [comment]
Its may be UI library bug

**Comment by jeanfbrito** [comment]
Not fully solved, sometimes when the tooltip of a "empty tooltip" apears with just an spance inside, bugs the interface again. This is driving me nuts... Haha

**Comment by oddstr13** [comment]
Recording of the issue appearing.
![recording](https://thoas.feralhosting.com/oddstr13/sharex/image/2017-04-26_05-27-43.gif)

Hovering the input bar breaks the UI.
Moving the ESPlorer window to the secondary monitor gives a working UI on that monitor, until the hover-text of the command input is displayed again. Moving back to main monitor does not fix the issue.

OS: Windows 10
GPU: GTX1070
Java:
```
java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
```

The following appears on the console, not sure if related:
```
[0x7FFEC09E5110] ANOMALY: meaningless REX prefix used
[0x7FFEB6D84250] ANOMALY: meaningless REX prefix used
```
